### DISCUSSION TITLE
Do you guys have any systems to close stale / long PRs?

### DISCUSSION THREAD

**Discussion by DanielMabbett** [gruntwork-io/knowledge-base#648] [Feedback]
I have been observing this PR for a while (for background, we really would like to see this feature in terragrunt to allow AzureRM backends to be automatically generated):
https://github.com/gruntwork-io/terragrunt/pull/1736

Just because this has been over a year open, trying to understand reasoning? Does it make sense to close stale PRs if this has become the case?

---

<ins datetime="2023-01-26T16:35:54Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109840">Tracked in ticket #109840</a></p>
</ins>

**Comment by zackproser** [comment]
Hi @DanielMabbett - thanks for the question, and apologies for the stale pull request. 

What I can share with you is that we're currently overhauling our process for managing the (great deal) of community pull requests we receive, so please watch this space for further updates in the coming weeks!

**Comment by gitsstewart** [comment]
@DanielMabbett as @zackproser mentioned, we are working on a better process for handling PRs in general, including what to do with old PRs. Right now we need to manage our most current PRs first but we will be moving through the older ones gradually. 
As to that specific PR, we are not currently supporting Azure. While we would like to do so in the future, right now AWS is our main focus and there is always a lot to keep us busy with AWS. 

The PR has been [updated](https://github.com/gruntwork-io/terragrunt/pull/1736#issuecomment-1412026680) to reflect that so others don't continue to wonder where we're at with it. 
Thanks for giving us feedback on your needs and preferences!
